### ISSUE TITLE
Latest news items are not always displayed

### ISSUE THREAD

**Issue by jszanto** [GEWIS/gewisweb#487]

**Comment by armandduijn** [comment]
Created a new design because the horizontal "news ticker" didn't work. What do you think? The arrows after the activity description are clickable and direct to the activity page.

![download](https://cloud.githubusercontent.com/assets/706385/17645478/68a2e370-61a7-11e6-8e1a-5c130c97ec05.png)

**Comment by jszanto** [comment]
@armandduijn I'm not sure where your comment went, but the new design looked pretty good. Although it does belong in another issue.

**Comment by armandduijn** [comment]
@jszanto Correct. Only after commenting I realised this was not a design issue but a functionality issue.

**Comment by jszanto** [comment]
Fixed by #697
